src/host_checks/CountriesContacts.cpp

```diff
@@ -0,0 +1,28 @@
+#include "ntop_includes.h"
+#include "host_checks_includes.h"
+
+CountriesContacts::CountriesContacts() : HostCheck(ntopng_edition_community, false /* All interfaces */, true /* Exclude for nEdge */, false /* NOT only for nEdge */) {
+    countries_contacts_threshold = (u_int32_t)100;
```

Comment:
default must be (u_intXX)-1 so that no threshold is triggered.

Reply:
fixed